homeassistant/components/bbb_gpio/__init__.py

```diff
@@ -11,7 +13,6 @@
 def setup(hass, config):
     """Set up the BeagleBone Black GPIO component."""
     # pylint: disable=import-error
-    from Adafruit_BBIO import GPIO
```

Comment:
Looks like this incarnation of the impl will leave a bunch of those `# pylint: disable=import-error`s behind; they should be cleaned up one way or another (either moved to end of the import line if still needed, or removed altogether if no longer necessary). Ditto possibly some `# noqa`s if there are any.